api/validators/UserControllerRequests.ts

```diff
@@ -29,6 +29,8 @@ export class UserPatches implements IUserPatches {
   @Allow()
   bio?: string;
 
+  resumeOptedIn?: boolean;
+
```

Comment:
When we say "opt-in", what are the users opting in to? Are they opting into recruiters viewing their resumes (i.e., only select accounts with RECRUITER types can see these, and opting out meaning no one can see their resume except themselves)? Or are they opting into their resumes being public? (i.e., all users can see their resumes, vs. opting out means only recruiters see the resumes)

Reply:
Also, let's rename to something like `isResumePublic` so that it's clearer what the resume is being opted into (depending on what it actually is being opted into)

Reply:
Opt-in in this case is referring to the ability of the recruiters to view the resume and consider the owner of the resume a potential recruit. This is necessary because there could be cases where a member puts their resume on the website without a need for the recruiters to see it, such as simply viewing their resume from other devices or already hired to a job.